content/aro/clf-to-azure-old/index.md

```diff
@@ -0,0 +1,145 @@
+---
```

Comment:
I think you have two copies of this file, one in a -old directory, one named -old.

Reply:
Yeah, I just fixed this. I didn't realize that you can't go directly to a named file in our publishing system, so I ended up having to create two directories and then clean up the dupes between them.